test/Microsoft.OpenApi.Tests/Models/References/OpenApiHeaderReferenceTests.cs

```diff
@@ -82,8 +82,8 @@ public class OpenApiHeaderReferenceTests
         public OpenApiHeaderReferenceTests()
         {
             OpenApiReaderRegistry.RegisterReader(OpenApiConstants.Yaml, new OpenApiYamlReader());
-            _openApiDoc = OpenApiDocument.Parse(OpenApi, OpenApiConstants.Yaml).OpenApiDocument;
-            _openApiDoc_2 = OpenApiDocument.Parse(OpenApi_2, OpenApiConstants.Yaml).OpenApiDocument;
+            _openApiDoc = OpenApiDocument.ParseAsync(OpenApi).GetAwaiter().GetResult().OpenApiDocument;
```

Comment:
this test and all the others below: switch to async lifetime, like the example I pushed for **OpenApiExampleReferenceTests**